src/Elmish.WPF/WpfProgram.fs

```diff
@@ -155,6 +158,17 @@ module WpfProgram =
     { program with LoggerFactory = loggerFactory }
 
 
+  /// Uses the specified handler for dispatch loop exceptions, dispatching the returned
+  /// messages when an exception occurs. The first (string) argument of onError is message
+  /// from Elmish describing the context of the exception, and may contain a rendered
+  /// message case.
+  ///
+  /// The update handler for the dispatched messages MUST NOT cause exceptions themselves,
+  /// or the app will enter into an infinite loop.
+  let withErrorHandler onError program =
```

Comment:
`onError` here has the signature `string * exn -> 'msg list`. Personally I would favor `string -> exn -> 'msg list`. The former is consistent with Elmish's own `withErrorHandler`, but that is not really relevant for Elmish.WPF users, since they don't use the Elmish.Program module directly. I suggest we change the signature of `onError` to `string -> exn -> 'msg list`.

Reply:
Yes, let use the curried version